.github/workflows/macos-ci.yml

```diff
@@ -12,7 +12,7 @@ env:
   poetry-version: 1.1.13
 
 jobs:
-  python-only:
+  macos-python-only:
```

Comment:
macos-ci-python-only ?

Reply:
And why "python-only" where other ci jobs are just "python"

Reply:
python-only because, it only test the python code not `python & rust` code